.github/workflows/publish.yaml

```diff
@@ -0,0 +1,91 @@
+name: Publish site and examples
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - packages/nuejs.org/**
+      - packages/examples/**
+
+concurrency:
+  group: 'pages'
+  cancel-in-progress: false
+
+jobs:
+  prepare-ssh:
+    if: ${{ github.repository_owner == 'nuejs'}}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Prepare SSH
```

Comment:
Maybe simpler to use https://github.com/marketplace/actions/setup-ssh ?

Reply:
Thanks